website/project/model.py

```diff
@@ -2756,7 +2755,7 @@ def remove_contributor(self, contributor, auth, log=True):
         self.save()
 
         #send signal to remove this user from project subscriptions
-        auth_signals.contributor_removed.send(contributor, node=self)
+        project_signals.contributor_removed.send(contributor, node=self)
```

Comment:
Listeners on the `auth` vs `project` signals had reversed `args` order, I picked one format arbitrarily. Trivial to update if you think it makes more sense the other way.